content/en/cloud_cost_management/realtime_costs.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Realtime Costs
+private: true
+description: View and analyze cloud spend in real time.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+---
+
+{{< callout url="#" btn_hidden="true" header="Realtime Costs is in Preview" >}}
+
+## Overview
+
+Realtime Costs provide near realtime estimates of your Amazon EC2 compute spend allowing you to see changes before they appear on your bill.
+
+Use realtime costs to:
+- Detect anomalies early
+- Observe the impact of recent changes
+- Monitor hourly or sub-hourly spend trends
+
+Estimates are based on recent average hourly net amortized EC2 prices by instance type, region, and aws account.
+
+## Requirements
+
+- Cloud Cost Management enabled for the AWS account
+- Datadog Agent installed on each EC2 instance
+
+## Usage
+
+To use realtime costs it is recommended to always use the `sum` aggregation. In addition a `rollup` using a `sum` with an interval no less than 5min is recommended. `sum:aws.cost.net.amortized.realtime.estimated.test{*}.as_count().rollup(sum, 300)`
+
+Using `count` shows the amount spent over the interval of time. For example if you set a 5min rollup, then the count is the amount spent in 5min. If the rollup is instead 1h it would be about 12 times as large.
+Using `rate` shows the amount spent per second. Using a formula to multiply this value can get you the rate per unit of time. For example taking the rate per second times 3600 will give you the rate per hour.
+
+You can learn more about rate metrics [here][1].
+
+## Accuracy
+
+- Sources of variance include:
+  - Hourly averages that shift with your recent mix of on-demand, commitment, and spot spend
+  - Minor differences in true instance start and end times vs what the agent reports
+- Underestimation can occur when:
+  - EC2 instances are not monitored by the Datadog Agent
+  - Newly used instance types or regions have not yet appeared in CCM billing data
+  - Estimates cover compute only (not EBS, networking, etc.)
+- Overestimation can occur when:
+  - Instances are monitored by the Datadog Agent but are not included in CCM billing data
+- Preview accuracy:
+  - During the preview period data may be unstable. Large temporary drops in cost or gaps may occur
+  - For long term trends it is still always recommended to use the accurate Cloud Cost metrics based on direct AWS billing data
+
+## Limitations
+
+Tagging may be inconsistent when compared to the other CCM metrics.
+Not all AWS billing or FinOps FOCUS tags are able to be replicated to the realtime cost metric.
+Realtime costs also do not support tag pipelines. Realtime costs do not support Kubernetes resource tags such as `kube_namespace`. Therefore only tags that show up on Amazon EC2 instance directly will be available. Tags that exist only on Kubernetes nodes will not be available.
```

Comment:
```suggestion
Realtime costs also do not support tag pipelines or Kubernetes resource tags such as `kube_namespace`. Only tags that appear on the Amazon EC2 instance directly are available. Tags that exist only on Kubernetes nodes are not available.
```

Reply:
This one says it is outdated, I think there were conflicting comments on this line 😨